packages/integrations/test/captureconsole.test.ts

```diff
@@ -0,0 +1,289 @@
+import { Event, Integration } from '@sentry/types';
+
+import { CaptureConsole } from '../src/captureconsole';
+
+const mockScope = {
+  setLevel: jest.fn(),
+  setExtra: jest.fn(),
+  addEventProcessor: jest.fn(),
+};
+
+const mockHub = {
+  withScope: jest.fn(callback => {
+    callback(mockScope);
+  }),
+  captureMessage: jest.fn(),
+  captureException: jest.fn(),
+};
+
+const getMockHubWithIntegration = (integration: Integration) => ({
+  ...mockHub,
+  getIntegration: jest.fn(() => integration),
+});
+
+// We're using this to un-monkey patch the console after each test.
+const originalConsole = Object.assign({}, global.console);
```

Comment:
As discussed in person, we will leave this as-is because `getGlobalObject` will always return `global` anyways, which is the [global namespace](https://nodejs.org/api/globals.html#globals_global) in node.